RxCocoa/Common/DelegateProxy.swift

```diff
@@ -142,17 +143,18 @@ open class DelegateProxy : _RXDelegateProxy {
     - returns: Observable sequence of arguments passed to `selector` method.
      */
     open func methodInvoked(_ selector: Selector) -> Observable<[Any]> {
+        MainScheduler.ensureExecutingOnScheduler()
```

Comment:
Cool, just remember that some guys had problems with `CoreBluetooth` because of this :)